### ISSUE TITLE
Vets-api terminal access for Derek Fong

### ISSUE THREAD

**Issue by dfong-adh** [department-of-veterans-affairs/va.gov-team#77522]
### Your Name

Derek Fong

### Your GitHub handle (terminal access is managed through GitHub)

@dfong-adh

### Team, Role, and Company of the target individual

Benefits Decision Reviews, Software Engineer, Ad Hoc

### Product Manager (PM) name and email

Sade Ragsdale, sade@coforma.io

### Product Owner (PO) name and email

Zach Gildfine, zachary.goldfine@va.gov

### COR Name (leave "None" if you listed an AWS user name above)

Other - please specify in 'Additional Notes'

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

None

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [X] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [ ] dev
- [X] staging
- [ ] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

I am requesting read-only production access for investigating backend errors/issues with benefit appeals forms.

COR: Keith Riley
VOR: Jeff Langworthy

EQIP Transmittal:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/151783381/8fd36d0b-c879-49ae-9803-b66162b79898)

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by bdech** [comment]
@dfong-adh, you've been granted access to `Production`, 
with access to Production limited to 6 months, expiring 09/07/24.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
@dfong-adh Your Vets-API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@dfong-adh Your Vets-API Terminal Production access has expired.

Would you like to request an extension? You have 48 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by bdech** [comment]
@dfong-adh Your Vets-API Terminal Production access has expired.

Would you like to request an extension? You have 24 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by dfong-adh** [comment]
Yes, please extend my Vets-API Terminal Production access. Thank you!

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@dfong-adh, your Production access has been extended for 6 months, expiring 03/07/25.

Please set a reminder for yourself to return to this Issue and request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1726235201797799)

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by GWright321** [comment]
@dfong-adh  Your Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your
access will be revoked and you will need to fill out a new request.

**Comment by dfong-adh** [comment]
Yes, please extend my Vets-API Terminal Production access. Thank you!

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@dfong-adh  your Production access has been extended for 6 months, expiring 9/10/25.

Please set a reminder for yourself to return to this Issue, reopen it, and request an extension within 1 week of the expiration date. You'll receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by bdech** [comment]
Support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1741724072688709)
